### DISCUSSION TITLE
Use different _app and _document for certain pages

### DISCUSSION THREAD

**Discussion by hirani89** [vercel/next.js#28407] [Help] [answered]
Hi,

I want to use wkhtltopdf to generate PDFs of the certain pages. I can not use the current pages as the source because the design requirement of the PDFs is different. Hence I want to create a different design which does not use the current _app and _document files.

**Comment by leerob** [comment, marked as answer]
Please read the docs on layouts, which can solve this for you 😄 

https://nextjs.org/docs/basic-features/layouts
